packages/components/src/templates/next/components/complex/LogoCloud/LogoCloud.tsx

```diff
@@ -34,7 +26,7 @@ export const LogoCloud = ({
             <ImageClient
               {...props}
               width="100%"
-              className="inset-0 w-fit max-w-52 object-contain p-2"
+              className="inset-0 max-h-24 w-fit object-contain p-2"
```

Comment:
do we need to verify chromatic for this pr?

Reply:
<img width="844" alt="image" src="https://github.com/user-attachments/assets/f2955544-b42b-4bb5-9190-f01aea5d0821" />

approved by our stand in designer